python/ray/autoscaler/updater.py

```diff
@@ -423,16 +423,15 @@ def do_update(self):
             # Run init commands
             self.provider.set_node_tags(
                 self.node_id, {TAG_RAY_NODE_STATUS: STATUS_SETTING_UP})
-            with LogTimer(self.log_prefix +
-                          "Initialization commands completed"):
+            with LogTimer(self.log_prefix + "Initialization commands", True):
```

Comment:
can you add the keyword label so by just looking at this piece of code we understand what's going on? `show_status=True`

Reply:
Good idea!